src/Stratis.Bitcoin.Features.MemoryPool/TxMemPoolEntry.cs

```diff
@@ -284,5 +293,34 @@ public override string ToString()
         {
             return $"{this.TransactionHash} - {base.ToString()}";
         }
+
+        /// <summary>
+        /// Default comparator for comparing this object to another TxMemPoolEntry object.
+        /// </summary>
+        /// <param name="other">Memory pool entry to compare to.</param>
+        /// <returns>Result of comparison function.</returns>
+        public int CompareTo(object other)
+        {
+            return uint256.Comparison(this.TransactionHash, (other as TxMempoolEntry).TransactionHash);
+        }
+
+        /// <summary>
+        /// Used to compare fees on objects supporting the IComparable and ITxMempoolFees interfaces.
+        /// </summary>
+        /// <typeparam name="T">A type that supports the IComparable and ITxMempoolFees interfaces.</typeparam>
+        /// <param name="a">The first object to compare.</param>
+        /// <param name="b">The second object to compare.</param>
+        /// <returns>Returns -1 if a less than b, 0 if a equals b, and 1 if a greater than b.</returns>
```

Comment:
this method does not have to be here right?

Reply:
Yes it's static so it can also go elsewhere. Since it's related to TxMemPoolEntry it should be ok here.

Reply:
ok sure fine